### ISSUE TITLE
Configuration deleted after update

### ISSUE THREAD

**Issue by droopycool** [jorenn92/Maintainerr#624]
I am pretty sure there is something I am doing wrong and not a bug, but I really don't seem to find what... Everything runs just perfect, no complains until whenever I update maintainerr. Then all configuration, rules, etc. are deleted and I must configure everything again.

Happens whenever I update Maintainerr

I excpect all the rules and configurations to stay after an update

**Device (please complete the following information):**
 - OS: Unraid Version: 6.11.5 
 - Running in docker_network

**Comment by jorenn92** [comment]
Hey, could you let me know where your mount is pointing to? If it's not directed to /opt/data, that could be the reason for the issue you're experiencing.

**Comment by droopycool** [comment]
Do you mean this one?

![image](https://github.com/jorenn92/Maintainerr/assets/87247848/491580c4-4b3e-4e12-a240-f0e0a7138178)

**Comment by jorenn92** [comment]
Yes, your container path should be /opt/data. I think this is a fault in the default Unraid community app config. 

Do you by any chance know where to report this? An external party made the Unraid version, but i don’t know who..

**Comment by droopycool** [comment]
> Yes, your container path should be /opt/data. I think this is a fault in the default Unraid community app config.
> 
> Do you by any chance know where to report this? An external party made the Unraid version, but i don’t know who..

This worked perfectly! Thanks!!!

The Unraid version I installed from IBRACORP's Repository https://ibracorp.io/
